src/main/java/jenkins/plugins/http_request/auth/BasicDigestAuthentication.java

```diff
@@ -64,6 +64,10 @@ public void authenticate(DefaultHttpClient client, HttpContext context,
     @Extension
     public static class BasicDigestAuthenticationDescriptor extends Descriptor<BasicDigestAuthentication> {
 
+        public String getHttpRequestHelpPath() {
+            return "/descriptor/"+HttpRequest.class.getName()+"/help";
```

Comment:
BTW I think you can use

```java
return "/" + Jenkins.get[Active]Instance().getDescriptor(HttpRequest.class).getDescriptorFullUrl() + "/help";
```

Be sure to check usage using `mvn hpi:run` which forces a nonempty context path (`/jenkins`).